Dockerfile

```diff
@@ -18,6 +18,7 @@ ENV FIREFOX_VERSION 58.0*
 ENV GECKO_DRIVER_VERSION v0.19.1
 ENV ROBOT_FRAMEWORK_VERSION 3.0.2
 ENV SELENIUM_LIBRARY_VERSION 3.0.1
+ENV FAKER_VERSION 4.2.0
```

Comment:
Just a detail but could you keep the ENV variables here in alphabetical order please? This will make it easier to find a specific line as the list grows.